### PR TITLE
[Core] Remove python 2 support from core

### PR DESCRIPTION
**Pull request by sunethwarna**
**Description**
As in the title this removes `from __future__ import` lines from python scripts.

**Changelog**
- Removes python 2 support from core

### CODE REVIEW

File: kratos/mpi/python_scripts/distributed_gid_output_process.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/mpi/python_scripts/distributed_gid_output_process.py]:
```suggestion
```
Please also remove  white lines

File: kratos/python_scripts/analysis_stage.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/analysis_stage.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/applicationGenerator.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/applicationGenerator.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/classes/classCreator.py [hunk 4]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/classes/classCreator.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/classes/classMemberCreator.py [hunk 5]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/classes/classMemberCreator.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/classes/conditionCreator.py [hunk 6]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/classes/conditionCreator.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/classes/elementCreator.py [hunk 7]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/classes/elementCreator.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/classes/processCreator.py [hunk 8]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/classes/processCreator.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/classes/variableCreator.py [hunk 9]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/classes/variableCreator.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/createApplication.py [hunk 10]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/createApplication.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/laplacian_application_example.py [hunk 11]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/laplacian_application_example.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/utils/io.py [hunk 12]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/utils/io.py]:
```suggestion
```

File: kratos/python_scripts/application_generator/utils/templateRule.py [hunk 13]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/utils/templateRule.py]:
```suggestion
```

File: kratos/python_scripts/base_convergence_criteria_factory.py [hunk 14]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/base_convergence_criteria_factory.py]:
```suggestion
```

File: kratos/python_scripts/fix_scalar_variable_process.py [hunk 15]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/fix_scalar_variable_process.py]:
```suggestion
```

File: kratos/python_scripts/fix_vector_variable_process.py [hunk 16]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/fix_vector_variable_process.py]:
```suggestion
```

File: kratos/python_scripts/gid_output_process.py [hunk 17]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/gid_output_process.py]:
```suggestion
```

File: kratos/python_scripts/integration_values_extrapolation_to_nodes_process.py [hunk 18]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/integration_values_extrapolation_to_nodes_process.py]:
```suggestion
```

File: kratos/python_scripts/kratos_collada.py [hunk 19]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/kratos_collada.py]:
```suggestion
```

File: kratos/python_scripts/kratos_utilities.py [hunk 20]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/kratos_utilities.py]:
```suggestion
```

File: kratos/python_scripts/line_output_process.py [hunk 21]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/line_output_process.py]:
```suggestion
```

File: kratos/python_scripts/modeler_factory.py [hunk 22]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/modeler_factory.py]:
```suggestion
```

File: kratos/python_scripts/multiple_points_output_process.py [hunk 23]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/multiple_points_output_process.py]:
```suggestion
```

File: kratos/python_scripts/new_linear_solver_factory.py [hunk 24]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/new_linear_solver_factory.py]:
```suggestion
```

File: kratos/python_scripts/print_vars_to_xml.py [hunk 25]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/print_vars_to_xml.py]:
```suggestion
```

File: kratos/python_scripts/process_factory.py [hunk 26]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/process_factory.py]:
```suggestion
```

File: kratos/python_scripts/python_linear_solver_factory.py [hunk 27]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/python_linear_solver_factory.py]:
```suggestion
```

File: kratos/python_scripts/read_materials_process.py [hunk 28]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/read_materials_process.py]:
```suggestion
```

File: kratos/python_scripts/scipy_conversion_tools.py [hunk 29]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/python_scripts/scipy_conversion_tools.py]:
```suggestion
```

File: kratos/python_scripts/skin_detection_process.py [hunk 30]
```diff
@@ -1,19 +1,18 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/skin_detection_process.py]:
```suggestion
```

File: kratos/python_scripts/sub_model_part_skin_detection_process.py [hunk 31]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/python_scripts/sub_model_part_skin_detection_process.py]:
```suggestion
```

File: kratos/tests/test_cad_json_input.py [hunk 32]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_cad_json_input.py]:
```suggestion
```

Comment (loumalouomega) [inline kratos/tests/test_cad_json_input.py]:
```suggestion
```

File: kratos/tests/test_calculate_distance_to_skin.py [hunk 33]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_calculate_distance_to_skin.py]:
```suggestion
```

File: kratos/tests/test_compare_elements_conditions.py [hunk 34]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_compare_elements_conditions.py]:
```suggestion
```

File: kratos/tests/test_compute_nodal_divergence.py [hunk 35]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_compute_nodal_divergence.py]:
```suggestion
```

File: kratos/tests/test_connectivity_preserve_modeler.py [hunk 36]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_connectivity_preserve_modeler.py]:
```suggestion
```

File: kratos/tests/test_dofs.py [hunk 37]
```diff
@@ -1,10 +1,9 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_dofs.py]:
```suggestion
```

File: kratos/tests/test_embedded_skin_mapping.py [hunk 38]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_embedded_skin_mapping.py]:
```suggestion
```

File: kratos/tests/test_exact_integration.py [hunk 39]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_exact_integration.py]:
```suggestion
```

File: kratos/tests/test_flags.py [hunk 40]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_flags.py]:
```suggestion
```

File: kratos/tests/test_geometries.py [hunk 41]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_geometries.py]:
```suggestion
```

File: kratos/tests/test_gid_io.py [hunk 42]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_gid_io.py]:
```suggestion
```

File: kratos/tests/test_gid_io_gauss_points.py [hunk 43]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_gid_io_gauss_points.py]:
```suggestion
```

File: kratos/tests/test_importing.py [hunk 44]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_importing.py]:
```suggestion
```

File: kratos/tests/test_kratos_parameters.py [hunk 45]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_kratos_parameters.py]:
```suggestion
```

File: kratos/tests/test_levelset_convection.py [hunk 46]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_levelset_convection.py]:
```suggestion
```

File: kratos/tests/test_linear_constraints.py [hunk 47]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_linear_constraints.py]:
```suggestion
```

File: kratos/tests/test_materials_input.py [hunk 48]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_materials_input.py]:
```suggestion
```

File: kratos/tests/test_matrix_interface.py [hunk 49]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_matrix_interface.py]:
```suggestion
```

File: kratos/tests/test_matrix_market_interface.py [hunk 50]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_matrix_market_interface.py]:
```suggestion
```

File: kratos/tests/test_model.py [hunk 51]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_model.py]:
```suggestion
```

File: kratos/tests/test_mortar_mapper.py [hunk 52]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (loumalouomega) [inline kratos/tests/test_mortar_mapper.py]:
```suggestion
```

File: kratos/tests/test_normal_utils.py [hunk 53]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_normal_utils.py]:
```suggestion
```

File: kratos/tests/test_numpy_export_dense_matrix.py [hunk 54]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_numpy_export_dense_matrix.py]:
```suggestion
```

File: kratos/tests/test_processes.py [hunk 55]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_processes.py]:
```suggestion
```

File: kratos/tests/test_numpy_export_dense_matrix.py (hunk 54 above)
Comment (loumalouomega) [inline kratos/tests/test_numpy_export_dense_matrix.py]:
```suggestion
```

File: kratos/tests/test_python_generic_function_utility.py [hunk 56]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_python_generic_function_utility.py]:
```suggestion
```

File: kratos/tests/test_redistance.py [hunk 57]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_redistance.py]:
```suggestion
```

File: kratos/tests/test_reorder.py [hunk 58]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_reorder.py]:
```suggestion
```

File: kratos/tests/test_restart.py [hunk 59]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_restart.py]:
```suggestion
```

File: kratos/tests/test_scipy_conversion_tools.py [hunk 60]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_scipy_conversion_tools.py]:
```suggestion
```

File: kratos/tests/test_serializer.py [hunk 61]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_serializer.py]:
```suggestion
```

File: kratos/tests/test_sparse_multiplication.py [hunk 62]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_sparse_multiplication.py]:
```suggestion
```

File: kratos/tests/test_tikz_output_process.py [hunk 63]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_tikz_output_process.py]:
```suggestion
```

File: kratos/tests/test_time_discretization.py [hunk 64]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_time_discretization.py]:
```suggestion
```

File: kratos/tests/test_variable_component.py [hunk 65]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_variable_component.py]:
```suggestion
```

File: kratos/tests/test_variable_utils.py [hunk 66]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_variable_utils.py]:
```suggestion
```

File: kratos/tests/test_vector_interface.py [hunk 67]
```diff
@@ -1,5 +1,4 @@
-﻿from __future__ import print_function, absolute_import, division
-
+﻿
```
Comment (loumalouomega) [inline kratos/tests/test_vector_interface.py]:
```suggestion
```

File: kratos/tests/test_vtk_output_process.py [hunk 68]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, absolute_import, division
```
Comment (loumalouomega) [inline kratos/tests/test_vtk_output_process.py]:
```suggestion
```

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Please remove whitespaces

**Comment by sunethwarna** [comment]
Removed whitespaces @loumalouomega  :)

**Review by loumalouomega — APPROVED** [review]
OK

**Pull request merged**